custom_components/adjustable_bed/__init__.py

```
@@ -1037,6 +1037,11 @@ async def handle_generate_support_bundle(call: ServiceCall) -> None:
                 address,
             )
         elif device_ids:
+            if len(device_ids) > 1:
+                raise ServiceValidationError(
+                    "Support bundle generation only supports one configured device at a time. "
+                    "Select a single device or use target_address for an unconfigured bed.",
+                )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)